README.md

```diff
@@ -1,13 +1,13 @@
-# Artifact Registry tools for npm
+# Artifact Registry tools for npm packages
 
 This repository contains tools to simplify the process of working with npm
-repositories using Artifact Registry.
+packages using Artifact Registry.
 
 # Artifact Registry Module
 
 The Artifact Registry google-artifactregistry-auth module is an npm package
-which allows you to configure npm to interact with npm repositories stored in
-Artifact Registry.
+which allows you to configure npm to interact with npm private repositories
+stored in Artifact Registry.
```

Comment:
Nit: I suggest removing "stored".

Reply:
Done. (Also maybe you can join https://github.com/orgs/GoogleCloudPlatform/teams/google-container-registry so that you have permission to "review" PRs?)